src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:hover {
+    transition-property: 300ms;
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+    color: $second-text-color;
+    margin-top: 4px;
+  }
+
+  &__description {
+    align-self: flex-start;
+    margin: 0 16px;
+  }
+
+  &__name {
+    padding-top: 40px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $main-text-color;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__review {
+    width: 166px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+
+    &-quantity {
+      font-size: 10px;
+      line-height: 14px;
+      color: $main-text-color;
```

Comment:
Ensure that the variable '$main-text-color' is defined in your SCSS files. If it's not defined, the color will not be applied correctly.